services/brig/src/Brig/IO/Intra.hs

```diff
@@ -172,9 +173,10 @@ updateSearchIndex orig e = case e of
           or
             [ isJust eupName,
               isJust eupAccentId,
-              isJust eupHandle
+              isJust eupHandle,
+              isJust eupManagedBy
```

Comment:
what about idp, role?

Reply:
@smatting Wanted to split this into two PRs I think. One for the general search endpoint and another one for sorting and filtering for these additional properties.

Reply:
idp: added a commit for that
role: not possible (easily) as this change to role originates in galley and I couldn't find an existing mechanism for creating `Event`s (`Brig.User.Event`) externally